contrib/objcwrapper/src/objc_visitor.nit

```diff
@@ -77,7 +77,7 @@ redef class Nsignature_block_signature
 		assert c != null
 
 		var method = new ObjcMethod(c)
-		method.return_type = n_signature_return_type.to_type
+		method.return_type = n_signature_return_type.as(not null).to_type
```

Comment:
This looks like a bug. Either the grammar is wrong or it should be handled better. Unless you want to learn the Objective-C grammar, you can leave the warning.